airbyte/_processors/sql/duckdb.py

```diff
@@ -93,9 +94,9 @@ def _write_files_to_new_table(
         columns_type_map = indent(
             "\n, ".join(
                 [
-                    f"{self._quote_identifier(c)}: "
-                    f"{self._get_sql_column_definitions(stream_name)[c]!s}"
-                    for c in columns_list
+                    f"{self._quote_identifier(prop_name)}: "
+                    f"{self._get_sql_column_definitions(stream_name)[prop_name.lower()]!s}"
```

Comment:
is there a reason we are not using the LowercaseNormalizer class here and doing .lower()?

Reply:
Good catch! No reason. There are probably a few more cases like this actually, where we can standardize the logic in one place.

An advantage of doing so will be that later, more edge cases can be all handled in one place - for instance, handling of punctuation and special characters.

Reply:
Fixed now. ✅